app/components/Reactions/ReactionPicker.js

```diff
@@ -175,13 +173,13 @@ const ReactionPicker = ({
     return searchEmojis(emojis, searchString);
   }, [searchString, emojis]);
 
-  const onCategoryClick = useCallback((category) => {
+  const onCategoryClick = (category) => {
```

Comment:
What was the warning/reason for this? I guess we should declare an empty array as deps, but not sure why it warns.

Reply:
This was caused by: ```warning  React Hook useCallback does nothing when called with only one argument. Did you forget to pass an array of dependencies?  react-hooks/exhaustive-deps` eslint-plugin-react-hooks```
Fixed this by passing an empty array as you suggested. :100: